Makefile

```diff
@@ -58,6 +59,7 @@ stage:
 docs.dev:
 	sphinx-build -b html -v docs docs/_build/
 
+# To be removed we don't really support older versions ATM anyway..
```

Comment:
We used to use [sphinx-multiversion](https://holzhaus.github.io/sphinx-multiversion/master/index.html) for building multiple versions based on tagging but it was always a finicky build.. was hoping that the next time we intended on working on the python-lib we could try and get it to work a little more like the certifai python docs @laguirre-cs